.gitignore

```diff
@@ -4,6 +4,7 @@
 .vs
 _ReSharper*/
 *.csproj.ReSharper
+.idea/
```

Comment:
I know single-line PRs can seem silly to some, but I would personally have preferred this to be a separate PR as it is completely unrelated to the rest of your proposed changes. (That said, it's a detail that's easily reviewed so I don't mind too much if it stays.)